src/Homepage/UpvoteDownvote.js

```diff
@@ -24,12 +25,16 @@ class UpvoteDownvote extends Component {
     this.state = {
       voteKey,
       voteValue: props.getItemFromStorage(voteKey),
-      upCount: props.entry.votes[VOTE.UP],
-      downCount: props.entry.votes[VOTE.DOWN],
+      voteCount: {
+        [UP]: props.entry.votes[UP],
+        [DOWN]: props.entry.votes[DOWN],
+      },
     };
 
     this.getVoteKey = this.getVoteKey.bind(this);
     this.handleVote = this.handleVote.bind(this);
+    this.handleUnselection = this.handleUnselection.bind(this);
+    this.handleNewSelection = this.handleNewSelection.bind(this);
   }
 
   getVoteKey(entry) {
```

Comment:
should this use `update id` and not store id?

Reply:
yes